src/main/java/th/ac/kmitl/science/comsci/example/controllers/HelloServlet.java

```diff
@@ -20,6 +20,9 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         response.setContentType("text/html");
         PrintWriter out = response.getWriter();
         out.println("<h1>" + message + "</h1>");
+	out.println("<ul>");
```

Comment:
Pull the upstream for "Bearchitect", "58050237 Chayapol Limanon", "57050294 Pichavarit Panyokaew" and "58050276 Thanathat Surakhup" to mege with your code